lib/constants.js

```diff
@@ -12,3 +12,14 @@ export const headers = {
 };
 
 export const didKeyDriver = didKey.driver();
+
+export const defaultHeaders = {
+  Accept: 'application/json, application/ld+json, */*'
+};
+
+export const postHeaders = {
+  'Content-Type': 'application/json',
+  ...defaultHeaders
+};
+
+export const sanitizeHeaders = ['Authorization', 'capability-invocation'];
```

Comment:
```suggestion
export const sanitizeHeaders = ['Authorization'];
```

Reply:
You don't need to sanitize capability invocations since they don't contain secrets like OAuth2 does.